client/me/privacy/style.scss

```diff
@@ -0,0 +1,10 @@
+.privacy__settings {
+    .form-toggle__switch {
```

Comment:
More of a minor note, but in many cases it's preferred to pass a classname to components to avoid excessive CSS specificity. 

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#classes

For example maybe:
```
.privacy__analytics-optout-switch {
    width: 44px;
    margin-top: 3px;
}
.privacy__analytics-optout-label {
    display: flex;
}
```

Reply:
Yep, in this case, I think the extra specificity isn't a big issue and using directly `form-toggle__label` / `form-toggle__switch` makes it easier to "fix" the flex issue directly in `FormToggle` as it would be easier to grok usages?

Reply:
We typically aim for a single class, to make our lives easier when we try to override defaults. So the [CSS specificity](https://gwwar.com/2015/09/10/why-didnt-that-selector-work/) score usually looks around (0,1,0), where we can override with another class. In this case It'd be (0,2,0). Minor, but more of an issue if this slips into a base component.